glib/src/object.rs

```diff
@@ -2199,6 +2200,22 @@ pub trait ObjectExt: ObjectType {
     /// Downgrade this object to a weak reference.
     fn downgrade(&self) -> WeakRef<Self>;
 
+    // rustdoc-stripper-ignore-next
+    /// Add a callback to be notified when the Object is disposed.
+    #[doc(alias = "g_object_weak_ref")]
+    #[doc(alias = "connect_drop")]
```

Comment:
I thought it might be nice to have something that someone might type into the search bar without any knowledge of glib. Maybe someone has a better idea for such a name.